src/tests/dbus-tests/test_80_filesystem.py

```diff
@@ -173,8 +173,29 @@ def test_repair_resize_check(self):
         # repair before resizing to half size, then check
         self.assertTrue(disk.Repair(self.no_options, dbus_interface=self.iface_prefix + '.Filesystem'))
         size = self.get_property(disk, '.Block', 'Size').value
+        self.get_property(disk, '.Filesystem', 'Size').assertEqual(size)
         disk.Resize(dbus.UInt64(size // 2), self.no_options, dbus_interface=self.iface_prefix + '.Filesystem')
         self.assertTrue(disk.Check(self.no_options, dbus_interface=self.iface_prefix + '.Filesystem'))
+        self.get_property(disk, '.Filesystem', 'Size').assertEqual(size // 2)
+
+    def test_size(self):
```

Comment:
Please skip this test for unsupported filesystems and filesystems that can't be created and/or mounted (see `_can_create` and `_can_mount` in other tests).

Also please "skip" this test for the `FailsystemTestCase`.

Reply:
Done, I think.

Reply:
No, not yet.  I forgot that we only report this for ext and xfs... that's what you meant with "unsupported", obviously.

Reply:
Should be done now.  There are test failures, but I think they are unrelated, no?

Reply:
Yes, the failures definitely look unrelated. Thanks.